src/main/java/com/untamedears/ItemExchange/utility/ItemExchange.java

```diff
@@ -316,77 +302,29 @@ public void cycleExchange(Player player) {
 		}
 		messagePlayer(player);
 	}
-	
-	public static void powerBlock(Player p, Block b) {
-		Material type = b.getType();
-		
-		if(!(type == Material.CHEST || type == Material.TRAPPED_CHEST || type == Material.FURNACE)) {
-			return;
-		}
-		
-		byte data = DeprecatedMethods.getBlockMeta(b);
-		
-		BlockFace face;
-		
-		if((data & 0x5) == 5) {
-			face = BlockFace.EAST;
-		}
-		else if((data & 0x4) == 4) {
-			face = BlockFace.WEST;
-		}
-		else if((data & 0x3) == 3) {
-			face = BlockFace.SOUTH;
-		}
-		else if((data & 0x2) == 2) {
-			face = BlockFace.NORTH;
-		}
-		else {
-			return;
-		}
-		
-		face = face.getOppositeFace();
-		
-		final Block b2 = b.getRelative(face, 2);
-		Material type2 = b2.getType();
-		
-		if(type2 == Material.STONE_BUTTON || type2 == Material.WOOD_BUTTON) {
-			BlockFace face2;
-			byte data2 = DeprecatedMethods.getBlockMeta(b2);
-			
-			if((data2 & 0x4) == 4) {
-				face2 = BlockFace.NORTH;
-			}
-			else if((data2 & 0x3) == 3) {
-				face2 = BlockFace.SOUTH;
-			}
-			else if((data2 & 0x2) == 2) {
-				face2 = BlockFace.WEST;
-			}
-			else if((data2 & 0x1) == 1) {
-				face2 = BlockFace.EAST;
-			}
-			else {
-				return;
-			}
-			
-			if(face2 == face) {
-				BlockState pressed = b2.getState();
-				DeprecatedMethods.setBlockMeta(pressed, (byte)(DeprecatedMethods.getBlockMeta(pressed) | 0x8));
-				pressed.update();
-				
-				Bukkit.getScheduler().scheduleSyncDelayedTask(ItemExchangePlugin.instance, new Runnable() {
-					public void run() {
-						BlockState pressed = b2.getState();
-						Material type = pressed.getType();
-						
-						if(!(type == Material.STONE_BUTTON || type == Material.WOOD_BUTTON)) {
-							return;
-						}
-						
-						DeprecatedMethods.setBlockMeta(pressed, (byte)(DeprecatedMethods.getBlockMeta(pressed) & ~0x8));
-						pressed.update();
-					}
-				}, 30);
+
+	public static void SuccessfulTransactionButton(Block shopChest) {
+		Material SC_Material = shopChest.getType();
```

Comment:
Variable name should start with lower case

Reply:
Done.